README.md

```diff
@@ -112,9 +112,11 @@ Please read more [here](https://docs.adyen.com/development-resources/client-side
 Use **Environment.test** for environment. When you're ready to accept live payments, change the value to one of our [live environments](https://adyen.github.io/adyen-ios/Docs/Structs/Environment.html)
 
 ```swift
-let apiContext = APIContext(clientKey: clientKey, environment: Environment.test)
-let context = AdyenContext(apiContext: apiContext, analyticsConfiguration: analyticsConfiguration)
-let configuration = DropInComponent.Configuration(context: context)
+let apiContext = try! APIContext(environment: componentsEnvironment, clientKey: clientKey)
```

Comment:
is it a good idea to use `automatic unwrapping` in the code snippet? 🤔

Reply:
No, but otherwise we need to do pseudo-code error handling (try/catch or guard) that is going to make snippet bigger 🤔

Do you have suggestions?

Reply:
hmm 🤔 , it sounds weird, but some merchants some copy past our example code and blame us when something wrong happens, its up to you :-)